Improvement of web proxy client code

Shokman opened the pull request:
This includes a merge of my previosu improvements with Pablo video work.
This includes:
- pep8 compilant code
- better ROS adapted code
- modified launch file used by concert_master

Shokman committed:
Improvement of web proxy client code

This includes a merge of my previosu improvements with Pablo video work.
This includes:

* pep8 compilant code
* better ROS adapted code

adamantivm commented on rosbridge_server/launch/rosbridge_websocket.launch:
```diff
@@ -7,7 +7,7 @@
   
   <group if="$(arg ssl)">
-    <node name="rosbridge_websocket" pkg="rosbridge_server" type="rosbridge_websocket" output="screen">
+    <node name="rosbridge_websocket_client" pkg="rosbridge_server" type="rosbridge_websocket_client" output="screen">
```
@Shokman Is this on purpose? This is going to break the regular websocket server!

Shokman committed:
Correct client and server launch files

rosbridge_websocket.launch and rosbridge_websocket_client.launch
modified to be running on the same machine form the same concert

Shokman commented:
@adamantivm please review changes

adamantivm commented on rosbridge_server/launch/rosbridge_websocket_client.launch:
```diff
@@ -4,6 +4,7 @@
   <arg name="certfile" />
   <arg name="keyfile" />
   <arg name="authenticate" default="false" />
+  <arg name="webserver_address" default="ws://localhost:9090" />
```
@Shokman  sorry to be annoying, can we please change this variable name to webserver_url or webserver_uri? I insist that the value is not an address but an URL (i.e.: it includes a protocol, a port and a path).
Also, please make the default ws://localhost:9090/ws to match with https://github.com/creativa77/rocon_web_proxy_server/blob/master/main.py#L97

Shokman committed:
Modify webserver_address to webserver_uri

It has been modified in both, launchfiles and websocket proxy client

Shokman commented:
@adamantivm refactoring done

adamantivm commented:
Thank you @Shokman :+1:
